app/models/miq_schedule_worker/miq_scheduler.rb

```diff
@@ -0,0 +1,29 @@
+class MiqScheduleWorker
+  class MiqScheduler
```

Comment:
We're in the MiqScheduleWorker namespace, it's probably ok to call this Scheduler.